src/btree/bt_split.c

```diff
@@ -1339,7 +1319,7 @@ __wt_split_insert(WT_SESSION_IMPL *session, WT_REF *ref, int *splitp)
 	 */
 	page = NULL;
 	if ((ret = __split_parent(session, ref, split_ref, 2,
-	    parent_decr, parent_incr, 0, 0, &split_gen)) != 0) {
+	    parent_decr, parent_incr, 0, &split_gen)) != 0) {
 		/*
```

Comment:
@michaelcahill, I don't yet understand this part of the change: obviously, the reason for `ref_discard` was to avoid calling `__split_ovfl_key_cleanup` on the original WT_REF's key, why is it safe to make that call?